Add better hint in the doctor for Maven workspaces (#744)
scalameta/metals: docs/build-tools/overview.md

## Goto library dependencies

> 

Reviewer: What do you think about replacing the checkboxes with words?

✅ becomes "Semi-automatic"

✔️ becomes "Automatic"

Author: We can do that, I was just thinking the idea was to use those ticks?

Reviewer: Seeing the table now, I'm not convinced which one of ✔️ and ✅ intuitively means more complete than the other. I think the words "Automatic" and "Semi-automatic" are more self-explanatory.

Author: Changed - let me know what you think

**Automatic**: it is possible to navigate Scala+Java library dependencies using "Goto
definition".

**Semi-automatic**: navigation in library dependency sources works as
long as the
[Bloop JSON files](https://scalacenter.github.io/bloop/docs/configuration-format/)
are populated with `*-sources.jar`.

## Find references

**Automatic**: it is possible to find all references to a symbol in the project

> **Semi-automatic**: it is possible to find references as soon as we configure additional SemanticDB plugin for the compiler, check separate build tool pages for details

Reviewer: ```suggestion
**Semi-automatic**: it is possible to 'Find symbol references' as soon the SemanticDB compiler plugin is manually enabled in the build, check separate build tool pages for details.
```
